src/odemis/gui/cont/tabs/sparc2_align_tab.py

```diff
@@ -1112,8 +1209,58 @@ def _onAlignMode(self, mode):
 
             self.panel.pnl_moi_settings.Show(False)
         elif mode == "light-in-align":
-            self.tab_data_model.focussedView.value = self.panel.vp_align_lens.view  # allows to see the focused slit line
-            self._ccd_stream.should_update.value = True
+            # FPLM specific codepath
+            if self.tab_data_model.fplm_module_present:
+                # For FPLM, the blanker should be enforced. Potentially also for other modes, which can be added later.
+                self.enforce_blanker()
+                if not self._ccd_stream.raw:
+                    logging.warning('Activate "LENS" mode first to display the CL Spot overlay stream')
+                else:
+                    # Eventhough the stream will be paused by now,
+                    # the raw should still contain the last obtained image,
+                    # due to order: detector pauses -> emitter pauses.
+                    self._ccd_stream_spot_snapshot.update(self._ccd_stream.raw[0])
+            self.tab_data_model.focussedView.value = self.panel.vp_align_light.view
+            self._ccd_stream_light.should_update.value = True
+
+            if self._mirror_settings_controller:
+                self._mirror_settings_controller.enable(False)
+            self.panel.pnl_mirror.Show(False)
+            self.panel.pnl_lens_mover.Show(False)
+            self.panel.pnl_lens_switch.Show(False)
+            if main.mirror:
+                self.panel.pnl_focus.Show(False)
+            else:
+                # If no mirror => ELIM => allow to focus, but only manually
+                self.panel.pnl_focus.Show(True)
+                self.panel.btn_autofocus.Enable(False)
+                self.panel.gauge_autofocus.Enable(False)
+            self.panel.pnl_focus_ext.Show(False)
+            self.panel.pnl_fibaligner.Show(False)
+            self.panel.pnl_streak.Show(False)
+            self.panel.pnl_light_aligner.Show(True)
+            self.panel.pnl_lens_tunnel.Show(False)
+            if main.spec_switch:
+                self.panel.pnl_spec_switch.Show(True)
+                self.panel.pnl_spec_switch.Enable(False)  # Wait until the spec-switch is engaged
+            else:
+                self.panel.pnl_spec_switch.Show(False)
+
+            self.panel.pnl_moi_settings.Show(False)
+            f.add_done_callback(self._on_light_in_align_done)
+        # This mode is only present for FPLM
+        elif mode == "light-in-align-ar":
+            self.enforce_blanker()
```

Comment:
Probably in every other modes, you need to unblank the beam, if it was blanked here. (Otherwise, the beam will stay blank all all modes).